.github/readmes/_using-the-graphql-api-typegraphql-crud.md

```diff
@@ -0,0 +1,139 @@
+## Using the GraphQL API
+
+`typegraphql-prisma` emits the generated TypeGraphql classes to `@generated/typegraphql-prisma` that is located in the `node_modules` whenever `npx prisma generate` is invoked. It also generates a number of model classes, enums and CRUD resolvers and relations resolver based on your `schema.prisma` file. CRUD resolvers support the following operations that are 1:1 matching with the `PrismaClient` API:
+
+- create
+- update
+- delete
+- findUnique
+- findFirst
+- findMany
+- updateMany
+- deleteMany
+- upsert
+- aggregate
+- groupBy
+
+Below are example operations that you can send to the API using the GraphQL Playground. You can explore other operations in the Docs section of the GraphQL Playground.
+
+Feel free to adjust any operation by adding or removing fields. The GraphQL Playground helps you with its auto-completion and query validation features.
+
+#### Retrieve all published posts and their authors
+
+```graphql
+query {
+  posts {
+    id
+    title
+    content
+    published
+    author {
+      id
+      name
+      email
+    }
+  }
+}
+```
+
+<Details><Summary><strong>See more API operations</strong></Summary>
+
+#### Create a new user
+
+```graphql
+mutation {
+  createUser(data: {
+    name: "Sarah",
+    email: "sarah@prisma.io"
```

Comment:
I think `sarah@prisma.io` already exists in the default DB in this case (should this be the case or did you accidentally upload a version of the dev.db that contains this record)? 

This leads to a violation of a uniquye constraint when running the query.